Library/Homebrew/cask/dsl/depends_on.rb

```diff
@@ -24,6 +24,7 @@ class DependsOn < SimpleDelegator
         intel:  { type: :intel, bits: 64 },
         # specific
         x86_64: { type: :intel, bits: 64 },
+        arm64: { type: :arm, bits: 64 },
```

Comment:
```suggestion
        x86_64: { type: :intel, bits: 64 },
        arm64:  { type: :arm, bits: 64 },
```

or `brew style --fix` should fix this.